pkgs/desktops/budgie/budgie-analogue-clock-applet/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, vala
+, budgie
```

We can probably be more specific here (the buildInput needs upd accordingly)
```suggestion
, budgie-desktop
```